grunt-configs/copy.js

```diff
@@ -31,11 +31,12 @@ module.exports = function (grunt, options) {
                         '**/*'
                     ],
                     dest: options.staticHashDir + 'javascripts/vendor'
-                },
-                {
+                },{
                     expand: true,
-                    cwd: options.requirejsDir,
-                    src: [
+                    cwd: options.isDev ? options.staticSrcDir + 'javascripts' : options.requirejsDir,
```

Comment:
Why is a verbatim copy needed from `static/src` to `static/target`? Aren't we loading directly from `static/src`?